weblate/checks/chars.py

```diff
@@ -100,6 +100,39 @@ def get_fixup(self, unit: Unit):
         return [("^ *", replacement, "u")]
 
 
+class KabyleCharacterNormalizationCheck(TargetCheck):
+    """Flag and suggest standard Kabyle characters instead of visually similar but incorrect ones."""
+
+    check_id = "kab_character_normalization"
+    name = gettext_lazy("Non‑standard characters in Kabyle")
+    description = gettext_lazy(
+        "Use standardized Latin Kabyle characters (e.g. ɣ instead of Greek γ; ɛ instead of ε)."
+    )
+
+    confusable_to_standard = {
+        "\u03b3": "\u0263",
+        "\u0393": "\u0194",
+        "\u03b5": "\u025b",
+        "\u0395": "\u0190",
+        "\u011f": "\u01e7",
+        "\u011e": "\u01e6",
+    }
+
+    def should_skip(self, unit: Unit) -> bool:
+        # Only run on Kabyle (covers 'kab' plus any variants)
+        return not unit.translation.language.is_base({"kab"})
```

Comment:
Keep the parent class logic in place as well:

```suggestion
        if not unit.translation.language.is_base({"kab"}):
           return True
        return super().should_skip(unit)
```